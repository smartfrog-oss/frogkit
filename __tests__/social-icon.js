import { mount } from 'vue-test-utils'
import SocialIcon from '@/components/social-icon'

const data = {
  url: 'http://www.facebook.com/smartfrogDE',
  name: 'facebook',
  label: 'Folgen Sie auf Facebook'
}

describe('Social Icon component', () => {
  it('Should render component and match snapshot', () => {
    const wrapper = mount(SocialIcon, { propsData: data })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
