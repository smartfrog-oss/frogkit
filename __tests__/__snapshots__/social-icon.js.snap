// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Social Icon component Should render component and match snapshot 1`] = `<a href="http://www.facebook.com/smartfrogDE" target="_blank" aria-label="facebook" rel="noopener" class="fk-social-icon facebook">Folgen Sie auf Facebook</a>`;
